packages/augur-ui/src/modules/market/components/market-header/market-header.tsx

```diff
@@ -190,7 +191,8 @@ export default class MarketHeader extends Component<
                   </button>
                   <MarketTypeLabel marketType={marketType} />
                 </WordTrail>
-                <div id="copy_marketId" data-clipboard-text={market.id}>
+                <SocialMediaButtons marketAddress={market.id} marketDescription={description} />
+                <div className={Styles.copy_marketId} id="copy_marketId" data-clipboard-text={market.id}>
```

Comment:
can class names be upper case? and also not sure if this needs its own class, could just do > div

Reply:
Yeah, I'll remove the class & update the CSS